README.md

```diff
@@ -28,10 +28,10 @@ to run), you will also need to install:
 
 If you want to use Qemu, we recommend the following steps:
 
-1) Create an Qemu image as a qcow2;
+1) Create an Qemu image in the qcow2 format;
```

Comment:
As we are already here, it would be nice to change "Create an Qemu [...]" for "Create a Qemu [...]".

Reply:
Yeah, it seems that the rule is the following: "Use “a” before words that start with a consonant sound and “an” before words that start with a vowel sound"

And Qemu indeed starts with a consonant sound.